github/repos_pages.go

```diff
@@ -62,6 +70,51 @@ func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo strin
 	}
 
 	return enable, resp, nil
+
+}
+
+type RemoveCName struct {
+	CName  *string `json:"cname"`
+	Source *string `json:"source,omitempty"`
+}
+
+func copyRemoveCName(opts *PageUpdate) *RemoveCName {
+	return &RemoveCName{
+		CName:  opts.CName,
```

Comment:
I think we should delete this line (83) because if `removeCustomDomain` (below) is true, we always want `CName` to be `nil`, and removing this line will enforce this behavior.